src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -33,6 +42,66 @@ public SealightsCLIBuildStep(boolean enabled, boolean failBuildIfStepFail,
         this.cliRunner = cliRunner;
     }
 
+    /* * The goal of this method is to support migration of data between versions
+    * of this plugin.
+    */
+    private Object readResolve() {
+        SealightsCLIBuildStep _this = this;
+        if (cliRunner != null) {
```

Comment:
This code is not clear.
Why do you return 'this'? Seems like 'resolveTechOption()' and 'resolveFromCLiRunner()' return this, so there is no need to override '_this' with the result of those methods.